build.sbt

```diff
@@ -60,49 +27,12 @@ Global / onLoad := (Global / onLoad).value.andThen { s =>
   s
 }
 
-// this overrides interplay release settings with some adaptations for working with sbt-dynver
-// moreover, it contain only bits necessary for twirl, nothing else
-lazy val releaseSettings: Seq[Setting[_]] = Seq(
-  // Release settings
-  releaseCrossBuild := false,
-  releaseProcess := {
-    import ReleaseTransformations._
-
-    def ifDefinedAndTrue(key: SettingKey[Boolean], step: State => State): State => State = { state =>
-      Project.extract(state).getOpt(key in ThisBuild) match {
-        case Some(true) => step(state)
-        case _          => state
-      }
-    }
-
-    Seq[ReleaseStep](
-      checkSnapshotDependencies,
-      runClean,
-      releaseStepCommandAndRemaining("+test"),
-      releaseStepTask(playBuildExtraTests in thisProjectRef.value),
-      releaseStepCommandAndRemaining("+publishSigned"),
-      releaseStepTask(playBuildExtraPublish in thisProjectRef.value),
-      ifDefinedAndTrue(playBuildPromoteBintray, releaseStepTask(bintrayRelease in thisProjectRef.value)),
-      ifDefinedAndTrue(playBuildPromoteSonatype, releaseStepCommand("sonatypeBundleRelease")),
-      pushChanges
-    )
-  }
-)
-
-val commonSettings = javaCompilerSettings ++ headerSettings ++ Seq(
-  scalaVersion := scala212,
-  crossScalaVersions := Seq(scala212, scala213),
-  scalacOptions ++= scalacCompilerParams,
-)
-
 lazy val twirl = project
   .in(file("."))
-  .enablePlugins(PlayRootProject)
+  .disablePlugins(MimaPlugin)
   .settings(
-    commonSettings,
-    releaseSettings,
     crossScalaVersions := Nil, // workaround so + uses project-defined variants
-    mimaFailOnNoPrevious := false
+    publish / skip := true
   )
   .aggregate(apiJvm, apiJs, parser, compiler, plugin)
```

Comment:
The `twirl` project doesn't declare (directly or via a plugin) a `scalaVersion`. It's probably fine since all projects get it declared via the `Common` plugin. It's only a bit surprising.

Reply:
I think it's fine since it's a aggregation project.